src/routes/InventoryTable.js

```diff
@@ -168,6 +171,61 @@ const Inventory = ({
 
     const calculateSelected = () => selected ? selected.size : 0;
 
+    //This wrapping of table actions allows to pass feature flag status and receive a prepared array of actions
+    const tableActions = (groupsUiStatus, row) => {
+        const isGroupPresentForThisRow = (row) => {
+            return row && row?.groups?.title !== '';
+        };
+
+        const standardActions = [
+            {
+                title: 'Edit',
+                onClick: (_event, _index, data) => {
+                    activateSystem(() => data);
+                    onEditOpen(() => true);
+                }
+            },
+            {
+                title: 'Delete',
+                onClick: (_event, _index, { id: systemId, display_name: displayName }) => {
+                    activateSystem(() => ({
+                        id: systemId,
+                        displayName
+                    }));
+                    handleModalToggle(() => true);
+                }
+            }
+        ];
+
+        const actionsBehindFeatureFlag = [
+            {
+                title: 'Add to group',
+                onClick: (_event, _index, { id: systemId, display_name: displayName }) => {
```

Comment:
`groupName` has to be assigned a value from the API; given that the preliminary field name must be `group_name`, we have to take the value from this field:

```suggestion
                onClick: (_event, _index, { id: systemId, display_name: displayName, group_name: groupName }) => {
```